scripts/distribution/ubuntu-packages/template/debian/concordium-node.service

```diff
@@ -28,27 +28,34 @@ LockPersonality=yes
 
 Environment=MODE=basic
 # port on which the node will listen for incoming connections
-Environment=LISTEN_PORT=8888
+Environment=CONCORDIUM_NODE_LISTEN_PORT=8888
 # where to bootstrap from
 Environment=BOOTSTRAP_FIRST_NODE=bootstrap.testnet.concordium.com:8888
 # desired number of nodes to be connected to.
-Environment=DESIRED_PEERS=5
+Environment=CONCORDIUM_NODE_CONNECTION_DESIRED_NODES=5
 # maximum number of __nodes__ the node will be connected to.
-Environment=MAX_ALLOWED_NODES=10
+Environment=CONCORDIUM_NODE_CONNECTION_MAX_ALLOWED_NODES=10
 # configuration and data directories.
-Environment=CONFIG_DIR=/var/lib/concordium/b6078154d6717e909ce0da4a45a25151b592824f31624b755900a74429e3073d/config
-Environment=DATA_DIR=/var/lib/concordium/b6078154d6717e909ce0da4a45a25151b592824f31624b755900a74429e3073d/data
+Environment=CONCORDIUM_NODE_CONFIG_DIR=/var/lib/concordium/b6078154d6717e909ce0da4a45a25151b592824f31624b755900a74429e3073d/config
+Environment=CONCORDIUM_NODE_DATA_DIR=/var/lib/concordium/b6078154d6717e909ce0da4a45a25151b592824f31624b755900a74429e3073d/data
 # address of the GRPC server
-Environment=RPC_SERVER_ADDR=0.0.0.0
+Environment=CONCORDIUM_NODE_RPC_SERVER_ADDR=0.0.0.0
 # and its port
-Environment=RPC_SERVER_PORT=10000
+Environment=CONCORDIUM_NODE_RPC_SERVER_PORT=10000
 # maximum number of __connections__ the node can have. This can temporarily be more than
 # the number of peers when incoming connections are processed. 
-Environment=HARD_CONNECTION_LIMIT=20
-# number of threads to use to process network events
-Environment=THREAD_POOL_SIZE=2
+Environment=CONCORDIUM_NODE_CONNECTION_HARD_CONNECTION_LIMIT=20
+# number of threads to use to process network events.
+Environment=CONCORDIUM_NODE_CONNECTION_THREAD_POOL_SIZE=2
 # Extra arguments that generally should not be altered.
-Environment=EXTRA_ARGS="--no-log-timestamp --bootstrapping-interval 1800 --no-dnssec --haskell-rts-flags=-N2"
+# Do not output timestamp in log output.
+Environment=CONCORDIUM_NODE_NO_LOG_TIMESTAMP=1
+# The bootstrapping interval in seconds.
+Environment=CONCORDIUM_NODE_CONNECTION_BOOTSTRAPPING_INTERVAL=1800
+# Do not perform DNSsec tests for lookups.
+Environment=CONCORDIUM_NODE_CONNECTION_NO_DNSSEC=1
```

Comment:
Wouldn't `true` be clearer and also work?

Reply:
I suppose.